Add settings page

Isak-Kallini opened the pull request:
Waiting for switch component

ManneG commented on src/routes/(app)/settings/+page.svelte:
```diff
@@ -1,5 +1,199 @@
+        onValueChange={(value) => setMode(value as "dark" | "light" | "system")}
+      >
+        <Select.Trigger class="w-36"
+          >{modeTranslation(data.theme)}</Select.Trigger
```
This only says "Dark" for me, even when selecting system or light (it changes the page successfully though)

<img width="215" height="128" alt="image" src="https://github.com/user-attachments/assets/bf883925-edbc-449c-bc8b-5df9ef642ce4" />

ManneG commented on src/routes/(app)/settings/+page.svelte:
One proposal for how it could work is, add a third "language option" being "no preference". This would be the default. If you have selected a preference it would (using a hook) always redirect you to the part of the site with you language of choice. This would then also hide the language button in the header as that would be redundant and do nothing.

Selecting english, and then going to `/news` would then redirect you to `/en/news` unless you went back to settings and selected no preference

Isak-Kallini commented:
Addressed comments. The language switching was broken on the settings page because of the select language dropdown and I didn't manage to fix it so I just commented it out for now so we can get the pr merged

Fiery-132 commented on biljett-bugg.dump:
pls remove
